### PR TITLE
add gcp-guest cluster into oss-prow build clusters

### PR DESCRIPTION
**Pull request by krzyzacy**
also added some dummy jobs to validate the secret

/assign @cjwagner 
cc @hopkiw @rofuentes

### CODE REVIEW

**Comment by cjwagner** [comment]
Oops this isn't `k/t-i`.
/lgtm

**Comment by krzyzacy** [comment]
```
ERROR: /home/prow/go/src/k8s.io/test-infra/WORKSPACE:30:1: Traceback (most recent call last):
	File "/home/prow/go/src/k8s.io/test-infra/WORKSPACE", line 30
		versions.check(minimum_bazel_version = "0.23.0")
	File "/root/.cache/bazel/_bazel_root/d14797137d3b356665173f69b0de28f1/external/bazel_skylib/lib/versions.bzl", line 106, in versions.check
		fail("Current Bazel version is {}; ex...))
Current Bazel version is 0.21.0; expected at least 0.23.0
ERROR: Error evaluating WORKSPACE file
ERROR: error loading package '': Encountered error while reading extension file 'go/deps.bzl': no such package '@io_bazel_rules_go//go': error loading package 'external': Could not load //external package
ERROR: error loading package '': Encountered error while reading extension file 'go/deps.bzl': no such package '@io_bazel_rules_go//go': error loading package 'external': Could not load //external package
INFO: Elapsed time: 16.720s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
ERROR: Build failed. Not running target
FAILED: Build did NOT complete successfully (0 packages loaded)
```

oops - obviously here everything is too outdated

**Comment by krzyzacy** [comment]
(updated kubekins and deployed config, trying again)

**Comment by krzyzacy** [comment]
hummm let me fix that job and bump prow here

File: prow/config.yaml [hunk 1]
```diff
@@ -71,17 +71,39 @@ presubmits:
...
-        - "&&"
-        - bazel
-        - run
-        - prow/cmd/checkconfig
-        - --
-        - --config-path=/home/prow/go/src/github.com/GoogleCloudPlatform/oss-test-infra/prow/config.yaml
+        - -c
+        - >-
+          cd /home/prow/go/src/k8s.io/test-infra &&
+          bazel run prow/cmd/checkconfig -- --config-path=/home/prow/go/src/github.com/GoogleCloudPlatform/oss-test-infra/prow/config.yaml
```
Comment (krzyzacy) [inline prow/config.yaml]: (rewrote the bash part here, not sure how it worked before :-\ )

**Comment by krzyzacy** [comment]
hummmm I also borked crier? /shrug

**Pull request merged**